### PR TITLE
Inject contentApiClient PR #2 (mostly in controllers)

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?

Inject contentApiClient in all necessary controllers (and one agent)
I am trying to keep the size of those PR manageable because it is quite repetitive at this point.
## What is the value of this and can you measure success?

➡️ Play 2.5

## Request for comment

@alexduf

### CODE REVIEW

File: common/app/common/dfp/TagSponsorship.scala [hunk 1]
```diff
@@ -141,30 +141,9 @@ case class PaidForTag(targetedName: String,
                       lineItems: Seq[GuLineItem]) {
 }
 
-object PaidForTag {
+object PaidForTagFormat {
```
Comment (alexduf) [inline common/app/common/dfp/TagSponsorship.scala]: I think this one should remain a companion object for `PaidForTag`

File: standalone/app/controllers/ItemController.scala [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+
 // If you add to this, don't forget the one in dev-build
-class ItemController(articleController: ArticleController, faciaDraftController: FaciaDraftController) extends ItemResponseController(
+class ItemController(articleController: ArticleController,
+                     faciaDraftController: FaciaDraftController,
+                     contentApiClient: ContentApiClient
+                    ) extends ItemResponseController(
   articleController,
-  GalleryController,
+  new GalleryController(contentApiClient),
```
Comment (alexduf) [inline standalone/app/controllers/ItemController.scala]: you could directly receive the controller as a parameter, as it is done for `articleController`, not that this has any sorts of impact as the controller shouldn't have any state

File: standalone/app/controllers/ItemController.scala [hunk 3]
```diff
@@ -10,7 +10,7 @@ class ItemController(articleController: ArticleController,
   articleController,
   new GalleryController(contentApiClient),
   MediaController,
-  InteractiveController,
+  new InteractiveController(contentApiClient),
```
Comment (alexduf) [inline standalone/app/controllers/ItemController.scala]: same here, could be passed, but not absolutely necessary

**Comment by alexduf** [comment]
All my comments are minor, that looks good :star: :+1: :star:

**Comment by TBonnin** [comment]
Thank you @alexduf. I have addressed you comments.

**Pull request merged**